### PR TITLE
List Ergo (ERG)

### PR DESCRIPTION
**Pull request by jasondavies**
- Official project URL: https://ergoplatform.org/
- Official block explorer URL: https://explorer.ergoplatform.com/

Example link to transaction: https://explorer.ergoplatform.com/en/transactions/fc9d78a3ca55d7950a928d55ef1676cc9ba516e4ef107d955de2fe541390ecc3

### CODE REVIEW

File: assets/src/main/java/bisq/asset/ErgoAddressValidator.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.Arrays;
+
+import org.bitcoinj.core.Base58;
+import org.bitcoinj.core.AddressFormatException;
+import org.bouncycastle.crypto.digests.Blake2bDigest;
+
+/**
+ * {@link AddressValidator} for Base58-encoded Ergo addresses.
+ */
+public class ErgoAddressValidator implements AddressValidator {
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/ErgoAddressValidator.java]: Please make this validator an inner class of Ergo class.

Comment (jasondavies) [reply assets/src/main/java/bisq/asset/ErgoAddressValidator.java]: Done!

**Review by blabno — APPROVED** [review]
ACK

**Comment by ripcurlx** [comment]
I'll merge this PR although Travis didn't run. It failed with a `Abuse detected` message (details are not disclosed). The build runs fine for me locally.

**Pull request merged**